src/email/create_campaign_analytics_dataset.R

```diff
@@ -0,0 +1,138 @@
+
+FP_CAMPAIGN_DATA <- "output/user_research/campaign_analytics_data.csv"
```

Comment:
small thing: maybe for consistency with the other files let's call the file "hdx_signals_campaign_analytics_data"?

Reply:
sure! 
52f49f0